sources/packages/web/src/assets/css/base.scss

```diff
@@ -79,15 +80,31 @@ html * {
   border-color: $badge-border-error !important;
 }
 
-/** Style to align records per page with arrows in data table. **/
+// Data table styles
+// Style to align records per page with arrows in data table.
 .p-paginator .p-dropdown .p-dropdown-label {
   margin-top: 8px;
 }
 
+.p-datatable.p-paginator-bottom {
```

Comment:
mm.. these are primevue data table classes. maybe we can add a todo, to remove these styles, when we move to vuetify data table

Reply:
@ann-aot I agree. I was thinking the same. You'll see I put all the p-datatable styles stuff under a comment, which we should be able to remove easily (and not think too much about it) when we change tables.

Reply:
@ann-aot Ah okay I see we have a todo style comment now. Thank you. Done!